core/spec/models/spree/payment_method/store_credit_spec.rb

```diff
@@ -269,9 +270,10 @@
                                         store_credit: store_credit)
       }
 
-      it "creates a store credit for the same amount that was captured" do
-        expect_any_instance_of(Spree::StoreCredit).to receive(:credit).with(captured_amount, auth_code, store_credit.currency)
-        subject
+      it "refunds the capture amount" do
+        expect { subject }.to change{ store_credit.reload.amount_remaining }.
```

Comment:
Although, there is no guidance in StemboltHQ/ruby-style-guide for these kinds of RSpec tests. At minimum, I would expect the spacing between `expect {` and `change{` to be consistent.